js/utils.js

```diff
@@ -1,21 +1,14 @@
 /* MagicMirror²
  * Utils
  *
- * By Rodrigo Ramírez Norambuena https://rodrigoramirez.com
+ * By KristjanESPERANTO https://github.com/KristjanESPERANTO
  * MIT Licensed.
  */
 const execSync = require("node:child_process").execSync;
-const colors = require("colors/safe");
 const Log = require("logger");
 const si = require("systeminformation");
 
 module.exports = {
-	colors: {
```

Comment:
would it be possible for a module developer to use Utils.color in his module? Just wondering, dont have time to test...

Reply:
Yeah, I think so. That would be an issue for those modules. I'll check tomorrow if any modules on the official list does that.

Reply:
Okay, I first searched all 1116 modules to see if they import `utils`. This is not the case :slightly_smiling_face:

Then I searched all modules to see if they use `colors`. I found one and it doesn't have it listed in its dependencies. For this one module, the change would therefore be relevant. So I have just created [a PR for the module's repository to fix this](https://github.com/djey47/MMM-IDF-LOCALTRANSPORT/pull/17).

Even if the PR for the module is not accepted, I would say that this is not a blocker for this PR here.